### PR TITLE
Add temporary rake task digest switch experiment

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/NvjAZOC8/454-get-the-data-needed-to-support-the-experiment

This is primarily so it can be retrieved by our PA on Integration.

### CODE REVIEW

File: lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Reports::SubscriptionChangesAfterSwitchToDailyDigest
+  def self.call
+    updated_subscriptions = Subscription.where(source: 4).where.not(ended_at: nil)
+
+    newer_subscriptions = updated_subscriptions.all.to_a.flat_map do |s|
+      Subscription
```
Comment (1pretz1) [inline lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb:6]: Could we do `Subscription.includes(:subscriber_list)` here and use the loaded relation to populate the subscriber list title on line 17? Slightly simplifies the code and means we can remove line 11.

Comment (1pretz1) [reply lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb:6]: Thinking about it don't know how well the above suggestion would scale

File: lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Reports::SubscriptionChangesAfterSwitchToDailyDigest
+  def self.call
+    updated_subscriptions = Subscription.where(source: 4).where.not(ended_at: nil)
+
+    newer_subscriptions = updated_subscriptions.all.to_a.flat_map do |s|
```
[the remark this answers is not in the document]
Comment (benthorner) [reply lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb:5]: Sure, I've removed that.

File: lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb (hunk 1 above)
Comment (benthorner) [reply lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb:6]: Yes we can/should 👍.

(I learnt about `index_by` recently, and it seems to have overridden my memory of `includes`!)

**Comment by benthorner** [comment]
Adding a "DO NOT MERGE" caveat until we're sure we can get access.

**Pull request merged**